### PR TITLE
New Datasource alicloud_hbr_backup_jobs; data_source/data_source_alicloud_hbr_snapshots: optimized property "status"and its docs; resource/alicloud_hbr_ecs_backup_client: optimize creation timeout and it's test case; resource/alicloud_hbr_ecs_backup_plan,resource/alicloud_hbr_nas_backup_plan,resource/alicloud_hbr_oss_backup_plan: adjust the property backup_type and it's document.

### PR DESCRIPTION
**Pull request by ChenHanZhang**
…ots property and its documents.

### CODE REVIEW

File: website/docs/d/hbr_backup_jobs.html.markdown [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	* `complete_time` -  The completion time of backup job. UNIX time seconds.
+	* `updated_time` - The update time of backup job. UNIX time seconds.
+	* `file_system_id` - The ID of destination file system.
+	* `nas_create_time` - File system creation time. UNIX time in seconds.
+	* `instance_id` - The ID of target ECS instance.
+	* `items_done` - The number of items restore job recovered.
+	* `items_total` - The total number of items restore job recovered.
+	* `paths` - Backup Path.
+	* `prefix` - Backup prefix.
+	* `include` - The include path.
```
Comment (xinmu0426) [inline website/docs/d/hbr_backup_jobs.html.markdown:77]: 补充下描述

File: alicloud/data_source_alicloud_hbr_snapshots.go [hunk 2]
```diff
@@ -81,7 +81,7 @@ func dataSourceAlicloudHbrSnapshots() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ForceNew:     true,
-				ValidateFunc: validation.StringInSlice([]string{"CANCELED", "COMPLETE", "QUEUED", "CANCELING", "FAILED", "PARTIAL_COMPLETE", "RUNNING", "UNKNOWN", "VM_UNAVAILABLE"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"COMPLETE", "PARTIAL_COMPLETE"}, false),
 			},
```
Comment (xinmu0426) [inline alicloud/data_source_alicloud_hbr_snapshots.go:85]: 修改状态定义是否正确？

File: website/docs/d/hbr_backup_jobs.html.markdown (hunk 1 above)
Comment (ChenHanZhang) [reply website/docs/d/hbr_backup_jobs.html.markdown]: 已补充

File: alicloud/data_source_alicloud_hbr_backup_jobs.go [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice([]string{"ECS_FILE", "NAS", "OSS"}, false),
+			},
+			"region_id": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
```
[the remark this answers is not in the document]
Comment (xiaozhu36) [reply alicloud/data_source_alicloud_hbr_backup_jobs.go:43]: 如果regionId是一个list，那么应该改成 region_ids

[the remark this answers is not in the document]
Comment (xiaozhu36) [reply alicloud/data_source_alicloud_hbr_backup_jobs.go:43]: 下面的参数也都一样，为啥是list，但是都不是ids

File: alicloud/data_source_alicloud_hbr_backup_jobs.go [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+			"Values":   v,
+			"Operator": "IN",
+		})
+	}
+	if v, ok := d.GetOk("vault_id"); ok {
+		filtersMapList = append(filtersMapList, map[string]interface{}{
+			"Key":      "VaultId",
+			"Values":   v,
+			"Operator": "IN",
+		})
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_hbr_backup_jobs.go:261]: 如果是这种kvpairs的形式，那么直接在schema中暴露一个 filters 就行，不需要把filter中的所有参数都单独暴露出来，具体的可以参考下：alicloud_express_connect_virtual_border_routers

File: alicloud/data_source_alicloud_hbr_ecs_backup_plans_test.go [hunk 5]
```diff
@@ -84,7 +84,7 @@ resource "alicloud_hbr_vault" "default" {
 }
 
 data "alicloud_instances" "default" {
-  name_regex = "hbr-ecs-backup-plan"
+  name_regex = "hbr-ecs-backup-plan-dont-delete"
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_hbr_ecs_backup_plans_test.go:87]: hbr-ecs-backup-plan-dont-delete 改成 no-deleteing-hbr-ecs-backup-plan

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Pull request merged**